terraform/main.tf

```diff
@@ -1,17 +1,75 @@
-// Here is where you should configure your remote state
+#Here you will configure your terraform and backend 
 terraform {
-  backend {}
-
-  required_providers {
-    aws = {
-      souce = "hashicorp/aws"
-    }
+  backend "s3" {
+    bucket  = "shift3-devopschallenge"
+    key     = "shift3-devopschallenge/terraform.tfstate"
```

Comment:
Might we need to mention that this should be renamed

Reply:
Good catch! It may not matter unless we have multiple applicants doing the challenge at the same time, but I think for configuration purposes this is a needed change.